app/global.css

```diff
@@ -11,6 +11,10 @@
   --primary-red: #db0a40;
 }
 
+body {
+  width: fit-content;
+}
```

Comment:
Cool find with fit-content; though I believe this is why the components do not span full screen width on screens larger than 1728px. 

![Screen Shot 2024-07-20 at 2 13 54 PM](https://github.com/user-attachments/assets/1715facc-878b-4c8b-9d29-14c0c1c7dd5d)

After some troubleshooting, I found that if you take this out completely and play with **SimpleGrid gap property** that you set to **lg: 200** (should be much lower than that) in Careers component (`line 82` in `Careers.tsx`), then both issues with the header not spanning full screen width starting at 992px and all the components not spanning as expected on screens >= 1728px go away. I guess that the card sizing in the `Careers component` combined with the gap size for screens >= 992px resulted in a width larger than the screen size and caused this bug with whitespace and horizontal scrolling.

Reply:
Yes, you were absolutely right on this one. Excellent debugging! I have no idea why I set that to 200 previously, but I brought it down to 90 and it now seems good.